stdlib/public/runtime/Metadata.cpp

```diff
@@ -3288,8 +3292,11 @@ static GenericWitnessTableCache &getCache(GenericWitnessTable *gen) {
                 sizeof(GenericWitnessTable::PrivateDataType),
                 "metadata cache is larger than the allowed space");
 
+  SWIFT_ALLOWED_RUNTIME_GLOBAL_CTOR_BEGIN
+  // FIXME -- Why is this Lazy<T> but not static?
```

Comment:
Can you file an SR for this as well?

Reply:
SR-7619

Reply:
The generic witness table's private data object is emitted by the compiler.

What warnings are you suppressing here?  Nothing about this code generates a global constructor, or for that matter a global object.

Reply:
Note that this code is basically assuming that Lazy<T> constant-initializes to a zero bit-pattern.